### ISSUE TITLE
NTR: Phosphorylase b kinase activity low in RBC & WBC (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8277]
New term request Phosphorylase b kinase activity low in RBC & WBC (elthat_110320173444): Erythrocytes and leukocytes, red and white blood cells

**Comment by pnrobinson** [comment]
[Phosphorylase](https://www.ncbi.nlm.nih.gov/books/NBK55061/)
 261750  PMID: 17689125 15/15

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide more context? Is is this enzyme (Phosphorylase kinase is a polymer of 16 subunits, four each of alpha, beta, gamma and delta, see - https://www.ncbi.nlm.nih.gov/gene/5256)

**Comment by MickeySegal** [comment]
Ours is indeed from https://www.ncbi.nlm.nih.gov/books/NBK55061/, which states "Phosphorylase kinase deficiency causing glycogen storage disease type IX (GSD IX) results from deficiency of the enzyme phosphorylase b kinase (PhK), an enzyme with a key regulatory role in the breakdown of glycogen. Deficiency of this enzyme, which is composed of four copies each of four subunits (α, β, γ, and δ), results in considerable clinical variability."

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000832
- label: Reduced tissue phosphorylase kinase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: 
- def: Concentration or activity of phosphorylase kinase below the lower limit of normal. This enzyme can be measured in multiple tissues including leukocytes and cultured fibroblasts.
- comment: Phosphorylase kinase is a polymer of 16 subunits, four each of alpha, beta, gamma and delta. The alpha subunit includes the skeletal muscle and hepatic isoforms, and the hepatic isoform is encoded by this gene. The beta subunit is the same in both the muscle and hepatic isoforms, and encoded by one gene. The gamma subunit also includes the skeletal muscle and hepatic isoforms, which are encoded by two different genes. The delta subunit is a calmodulin and can be encoded by three different genes. The gamma subunits contain the active site of the enzyme, whereas the alpha and beta subunits have regulatory functions controlled by phosphorylation. The delta subunit mediates the dependence of the enzyme on calcium concentration.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8277
- synonyms:
